docs/Model.md

```diff
@@ -132,6 +132,66 @@ Pass the _table name_ of the related records as an argument to `@children`. The
 
 **Note:** You must define a `has_many` association in `static associations` for this to work
 
+### Many-To-Many Relation
```

Comment:
I suggest moving it down to the "Advanced" section

Reply:
> > Also, I think we should migrate the relation docs from model docs to Relation docs for better visibility. If this okay I can go ahead and do that.
> 
> Ah, you're right. I think the very basics ( @children, relation) should stay as is, but the advanced details (like many-to-many ) should be moved to Relation.md

Yes, I think this is much better. I will move the docs there then.